#169589914 : Set up Winston logger

Mastel22 opened the pull request:
#### What does this PR do?
Sets up the winston logger for the project
#### Description of Task to be completed?

- [X] install winston package
- [X] create own logger

#### How should this be manually tested?

- Git clone this repo
- run `npm install`
- run `npm run winston`

#### Any background context you want to provide?
-N/A
#### What are the relevant pivotal tracker stories?
[#169589914](https://www.pivotaltracker.com/story/show/169589914)
#### Questions:
None

Commit:
chore(winston logger): Set up winston logger

- installed winston package
- created own logger

[finishes #169589914]

timolinn commented on src/utils/winston.js:
```diff
@@ -0,0 +1,18 @@
+  ),
+  transports: [
+    new transports.Console(),
+    new transports.File({ filename: './error.log', level: 'error' }),
```
Really good job here Mastel. We need a `logs` folder at the project root which would contain both files. The log files should be `.gitignore`d, once that is done we need a way to initialise the log files automatically. I suggest we write a short script to create these log files when they don't exist yet.

Mastel22 replied:
Okay. Let me work on that now

b0nbon1 commented on src/utils/winston.js:
```diff
@@ -0,0 +1,19 @@
+  transports: [
+    new transports.Console(),
+    new transports.File({ filename: '.logs/error.log', level: 'error' }),
+    new transports.File({ filename: './logs/info.log', level: 'info' }),
```
Great job but I have tested it but the error log creates the  `.logs/error.log` which works okay but the info does not so I suggest maybe the line `./logs/info.log` to `.logs/info.log` for it to work.

b0nbon1 replied:
And also on the pr description how to manually  you should add the script to run so as to test it

Mastel22 replied:
Oh yeah! Missed that. Fixed now. Thanks

Commit:
chore(winston logger): Fixing bugs

- Fixed the info.log file bug
- changed the format of the logger and console display

b0nbon1 reviewed (dismissed):
Great job, keep it up.

nshaphan reviewed (dismissed):
great work!
